.editorconfig

```diff
@@ -0,0 +1,30 @@
+# http://editorconfig.org
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+
+[*.{html,json}]
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = false
+
+[*.{scss,js}]
+indent_style = tab
+indent_size = tab
+
+
+# Customizations for third party libraries
+
+[static/scss/{font-awesome/*.scss,_font-awesome.scss}]
+indent_style = space
+indent_size = 2
+
+[static/js/lib/**]
```

Comment:
Better leave it as `[static/js/lib/**]`. In future we might have custom JS code. I will move the JS libs to a libs folder.